javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+let countdown = 4;
+
+function decrement() {
+  document.querySelector('.countdown-display').innerHTML = countdown;
```

Comment:
When you page loads it waits 1 second to call `decrement()`. At that point you are setting:

```
document.querySelector('.countdown-display').innerHTML = countdown;
```

This means after one second the displayed value of 4 remains 4. The, after one second (a total of two seconds) we change the value to 3. 

So you are experiencing a one second delay. 

To fix this decrement `countdown--` before this line of code.